clear-containers.service.in

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Clear Containers Agent
+Documentation=https://github.com/clearcontainers/agent
+
+[Service]
+ExecStart=@bindir@/@ccagent@
```

Comment:
@jcvenegas @sboeuf do you not need to restart on failure? In the past we used to reboot on hyperstart crash. How will we handle agent crash